ocaml/testsuite/tests/templates/basic/bad_ref_direct_imported.reference

```diff
@@ -0,0 +1,3 @@
+File "bad_ref_direct_imported.ml", line 1:
+Error: The file "monoid.cmi" contains the interface of a parameter.
+       "Monoid" is not declared as a parameter for the current unit (-parameter "Monoid").
```

Comment:
I think in this case, we _definitely_ want the error message to point at the exact location? Because the error will be received by end users.

Also how about:
> Error: The module `Monoid` is a parameter but is not declared as such for the current module.
> Hint: Compile the current unit with "-parameter Monoid"